app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationController < ActionController::Base
   private
 
   def skip_pundit?
+    devise_controller? || params[:controller] =~ /(^(rails_)?admin)|(^pages$)|(^developer_skills$)|(favourites)/
```

Comment:
Best not to do this, keep it all the same line. I'll sort it my self though in the next PR